JDRF/spirit
# Add watermarks to library

Add missing brushes to library

JIRA ticket: https://jdrf.atlassian.net/browse/DGST-45

[Abstract file](https://app.abstract.com/projects/e8a6a8d0-cf02-11e7-97e7-85865a55282a/branches/master/commits/3e087e3bfb5849b05a920f063e5f3194321d9fa5/files/4B5B141B-AD52-4BB2-AA44-3DA3263F261E/layers/1E6F160F-DFE3-4BFA-8965-8147440999E9?mode=design)

To test:
- Pull branch
- In `/library` run `gulp`
- open `/sink-pages/components/brushes.html` and verify brushes output

Next steps:
1. Update documentation in confluence [Doc]
2. Publish documentation in doc-site [Pub]
